### DISCUSSION TITLE
Support for MRC/CCP4 2014 file format

### DISCUSSION THREAD

**Discussion by lglass** [ImageMagick/ImageMagick#6977] [Contribute]
### Describe the solution you'd like

[MRC](https://www.ccpem.ac.uk/mrc_format/mrc2014.php) is a standard format for a certain kind of scientific data. It contains either 2d images or stacks of 2d images. It would be great to be able to read it with ImageMagick and write it to png for 2d or gif for "3d" images.

### Describe alternatives you've considered

[Fiji](https://imagej.net/software/fiji/) can read MRC and save it as something else. It depends mainly on a user interface and possibly macros.

[mrcfile](https://pypi.org/project/mrcfile/) in conjunction with Pillow would be the Python way to do it, I think.

**Comment by fmw42** [comment]
Sounds like an interesting file format. However, I suspect that IM does not have any mechanism for 3D files other than just different layers/pages.
